next-on-netlify.js

```diff
@@ -1,17 +1,26 @@
 #!/usr/bin/env node
 const { program } = require("commander");
 
-program
-  .option(
-    "--max-log-lines [number]",
-    "lines of build output to show for each section",
-    50
-  )
-  .parse(process.argv);
-
 const nextOnNetlify = require("./index");
-const { logTitle } = require("./lib/helpers/logger");
 
-nextOnNetlify();
+program.option(
+  "--max-log-lines [number]",
+  "lines of build output to show for each section",
+  50
+);
+
+program
+  .command("watch")
+  .description("re-runs next-on-netlify on changes")
+  .action(() => {
+    nextOnNetlify({ watch: true });
+  });
+
+program
+  .command("build", { isDefault: true })
```

Comment:
what is the isDefault flag for?

Reply:
oh, like, this is the default command of the program?

Reply:
Yep, it is there to support the existing use case of running `next-on-netlify` without any arguments